schema.graphql

```diff
@@ -174,3 +174,192 @@ type InterestRateModelSet @entity {
   floatingCurveB: BigInt!
   floatingMaxUtilization: BigInt!
 }
+
+type Account @entity {
+  id: ID! # address-market
+  address: Bytes!
+  market: Bytes!
+  borrowShares: BigInt!
+  depositShares: BigInt!
+  isCollateral: Boolean
+  fixedPositions: [FixedPosition!]! @derivedFrom(field: "account")
+}
```

Comment:
This looks more like a `UserPosition` than an `Account` entity

Reply:
`Account` is a better name, it works even for abstracted accounts like a multisig account.

Reply:
I think having an `Account` entity with floating position attributes like `isCollateral` is a bit misleading.